src/Component/Symbolizer/BulkEditModals/BulkEditModals.tsx

```diff
@@ -0,0 +1,180 @@
+import * as React from 'react';
+
+const _isEqual = require('lodash/isEqual');
+
+import {
+  Modal
+} from 'antd';
+
+import {
+  Style as GsStyle,
+  Rule as GsRule
+} from 'geostyler-style';
+
+import ColorField from '../../Symbolizer/Field/ColorField/ColorField';
+
+import { localize } from '../../LocaleWrapper/LocaleWrapper';
+import en_US from '../../../locale/en_US';
+import RadiusField from '../../Symbolizer/Field/RadiusField/RadiusField';
+import OpacityField from '../../Symbolizer/Field/OpacityField/OpacityField';
+
+// i18n
+export interface StyleLocale {
+  colorLabel: string;
+  radiusLabel: string;
+  opacityLabel: string;
+}
+
+// default props
+interface BulkEditModalsDefaultProps {
+  colorModalVisible: boolean;
+  sizeModalVisible: boolean;
+  opacityModalVisible: boolean;
+  style: GsStyle;
+  locale: StyleLocale;
+  compact: boolean;
```

Comment:
What's the purpose of this property? It looks like it's not being used.

Reply:
True, I removed all references in 8a6bb9b